### PR TITLE
Update warning message about enhancement on the repo

### PR DESCRIPTION
**Pull request by alallema**
As discussed with the team, this repo will no longer be open to enhancement. We think it's better to specify this in the README for greater clarity with contributors.

### CODE REVIEW

**Review by curquiza — COMMENTED** [review]
Can we talk about the Cloud alternative?

**Comment by alallema** [comment]
> Can we talk about the Cloud alternative?

I don't add it in this repo because it's already said [here](https://github.com/meilisearch/docs-scraper#-supercharge-your-meilisearch-experience), But I could add the sea sentence as in the meilisearch migration.

**Comment by curquiza** [comment]
It's more here to say there is alternative to this repo in our Cloud. No matter is the sentence, but we are missing this information in the README 😊

**Comment by alallema** [comment]
Yes the crawler! I add it

File: README.md [hunk 1]
```diff
@@ -30,6 +30,8 @@
 
 *Our team is small, and our availability will be reduced in the upcoming times. As such, response times might be slower, and we will not be accepting enhancements for this repository moving forward.*
 
+*If you're looking for reliable alternatives, consider using [Cloud Service](https://www.meilisearch.com/pricing?utm_campaign=oss&utm_source=integration&utm_medium=meilisearch-migration). It offers a robust solution for those seeking an alternative to this repository, and also features a crawler for your convenience.*
```
Comment (curquiza) [inline README.md:33]:
```suggestion
*If you're looking for reliable alternatives, consider using [Cloud Service](https://www.meilisearch.com/pricing?utm_campaign=oss&utm_source=integration&utm_medium=meilisearch-migration). It offers a robust solution for those seeking an alternative to this repository by providing a crawler for your convenience.*
```

Comment (curquiza) [reply README.md:33]: Why "and also"? For me the alternative is the crawler, right?

Comment (alallema) [reply README.md:33]: I think you're right, I was thinking of an alternative to meilisearch for the Cloud but this is much better as you write.

**Comment by alallema** [comment]
bors merge

**Pull request merged**